auto_nag/tests/test_query_creator.py

```diff
@@ -1,16 +1,25 @@
-
-from auto_nag.bugzilla.utils import os
+from auto_nag.common import get_current_versions
 from auto_nag.bugzilla.utils import get_project_root_path
 from auto_nag.scripts.query_creator import (getTemplateValue, getReportURL,
                                             createQueriesList, cleanUp, urls,
                                             esr_version)
-import shutil
-import datetime
+
 
 class TestQueryCreator:
     def setUp(self):
         self.queries_dir = get_project_root_path() + 'queries/'
 
+    def test_0_get_current_versions(self):
+        versions = get_current_versions()
+        nightly_version = versions["central"]
+        assert type(int(nightly_version)) is not type(int)
+        beta_version = versions["beta"]
+        assert type(int(beta_version)) is not type(int)
+        release_version = versions["release"]
+        assert type(int(release_version)) is not type(int)
+        esr_version = versions["esr"]
+        assert type(int(esr_version)) is not type(int)
```

Comment:
I'm having a hard time figuring out what this is actually testing.

type(int(foo)) is going to be int (or long.  or it's going to raise).  type(int) is type.  Those are never going to be the same thing.  What am I missing?

Reply:
that the keys exists and they are returning something :)
what would you prefer?